app/src/main/kotlin/com/intive/picover/common/di/AppModule.kt

```diff
@@ -19,4 +20,11 @@ object AppModule {
 	@Provides
 	fun provideFirebaseDatabase() =
 		Firebase.database
+
+	@Provides
+	fun provideShortTextValidatorFactory() =
```

Comment:
`provideShortTextValidatorFactory` but actually you are providing `TextValidator` which could be used for both short and long text due to `validatingText` `isLongText` parameter.

Reply:
updated